source/vibe/web/common.d

```diff
@@ -305,8 +321,10 @@ MethodAttribute method(HTTPMethod data)
 	
 	shared static this()
 	{
+		// Tie IAPI.getInfo to "GET /root/foo/info2"
 		registerRestInterface!IAPI(new URLRouter(), new API(), "/root/");
-		// now IAPI.getInfo is tied to "GET /root/info2"
+		// Or just to "GET /foo/info2"
```

Comment:
No sorry, you are right, I've skimmed to fast over this and mistook the following line for something else. Please ignore.